core/src/main/java/apoc/meta/tablesforlabels/PropertyTracker.java

```diff
@@ -39,7 +39,7 @@ private String assignTypeName(Object value) {
         return ret;
     }
 
-    private static final Map<String,String> typeMappings = new HashMap<String,String>();
+    public static final Map<String,String> typeMappings = new HashMap<String,String>();
```

Comment:
why did this change from private to public?

Reply:
Because I used the `typeMapping` to create the `REVERSED_TYPE_MAP` in `MetaInformation.java` 
(which, in fact, does the inverse of the `typeMapping`, that is, it gets `Class` from `propertyType`)